provisioning/windows-provision.ps1

```diff
@@ -250,14 +250,23 @@ $downloads = [ordered]@{
             & Remove-Item -Force -Recurse "$baseDir\chocolatey.tmp";
         };
         'cleanupLocal' = 'true';
-        'path' = "$baseDir\cygwin\bin\";
+        'path' = "$baseDir\ruby26\bin\;C:\HashiCorp\Vagrant\";
         'postInstall' = {
             # Installation of make for Windows
             & "choco.exe" install make --yes --no-progress --limit-output --fail-on-error-output;
+            & "choco.exe" install vagrant --yes --no-progress --limit-output --fail-on-error-output --version "${env:VAGRANT_VERSION}";
+            # Append a ".1" as all ruby packages in chocolatey have this suffix. Not sure why (maybe a package build id)
+            & "choco.exe" install ruby --yes --no-progress --limit-output --fail-on-error-output --version "${env:RUBY_VERSION}.1";
+            & "choco.exe" install yq --yes --no-progress --limit-output --fail-on-error-output --version "${env:YQ_VERSION}";
+            
         };
         'sanityCheck'= {
             & "choco.exe";
             & "make.exe" -version;
+            & "vagrant.exe" --version;

```

Comment:
```suggestion
            # & "vagrant.exe" --version;
```
I don't have a Windows VM to check where exactly vagrant is installed (maybe a sub directory of C:\HashiCorp\Vagrant\), commenting this added sanity check